apps/marginfi-v2-ui/src/components/common/Portfolio/hooks/use-reward-simulation.tsx

```diff
@@ -56,13 +102,13 @@ export const useRewardSimulation = ({
 
       const atas: PublicKey[] = [];
 
-      for (let bankAddress of bankAddressesWithEmissions) {
-        const bank = marginfiClient.getBankByPk(bankAddress);
-        if (!bank) continue;
+      for (let bank of banksWithEmissions) {
```

Comment:
Yes, makes sense. 

-> following this; since ata is always defined but the address isnt initialised, we also need to add instructions to create/initialise the ata right? I dont think we're doing that atm

Reply:
yeah we def do if the ata doesn't exist